docs/network/nodes.html

```diff
@@ -1071,6 +1072,14 @@ <h3>Options</h3>
                 drawn behind it and it has a border. This means all border options are taken into account.
             </td>
         </tr>
+        <tr parent="shapeProperties" class="hidden">
+            <td class="indent">shapeProperties.useImageCoords</td>
+            <td>Boolean</td>
+            <td><code>false</code></td>
+            <td>This property only applies to the <code>image</code> and <code>circularImage</code> shapes.
+                When true, the image's (0,0) coords are used for drawing.
```

Comment:
I have no idea what this means. Used for drawing? How? Why? What happens if it's not true?

Reply:
I'm really not sure how to describe the drawing process.  Currently, when an image/svg is used, the (x, y) of the node will be the center of the image (using width and height / 2). When this prop is set to true, the behaviour changes so the (x, y) of the node matches to the (0, 0) of the image.  Can you propose a concise description for this behaviour in a few words?

Reply:
I suggest some polished version of:
> When an image/svg is used, the (x, y) of the node will be the center of the image (using width and height / 2). When this prop is set to true, the behaviour changes so the (x, y) of the node matches to the (0, 0) of the image.